docs/kubernetes/ingress-controller.md

```diff
@@ -605,3 +605,121 @@ from inside a container:
 ```
 curl demo.default.skipper.cluster.local
 ```
+
+## Running with Cluster Ratelimits
+
+Cluster ratelimits require a communication exchange method to build a
+skipper swarm to have a shared knowledge about the requests passing
+all skipper instances. To enable this feature you need to add command
+line option `-enable-swarm` and `-enable-ratelimits`.
```

Comment:
How about changing the configuration concept to explicitly stating which rate limiter to use, i.e. `redis`, or `swim` and moving the `-enable-swarm` to the swim rate limiter description? (In my opinion for redis, there is no swarm).

Reply:
We thought about swarm being a general concept of shared data of skipper instances.
With this in mind you can do a redis based swarm, too.
I like the idea of having a switch, maybe drop the `-enable-swarm` and use `-swarm=<redis,swim>`.

Reply:
I would not relate the swarm metaphor for the redis implementation. This is a kind of centralized coordinator, even if it is distributed itself. This is why I would entangle this.